aiohttp/cookiejar.py

```diff
@@ -102,12 +110,16 @@ def _do_expiration(self) -> None:
         for key in to_del:
             del expirations[key]
 
-        self._next_expiration = ceil(next_expiration)
+        try:
+            self._next_expiration = (next_expiration.replace(microsecond=0) +
+                                     datetime.timedelta(seconds=1))
+        except OverflowError:
+            self._next_expiration = self.MAX_TIME
```

Comment:
I like how you handled the possible overflow here :-)

Reply:
Thanks.